packages/frontend/src/mantineTheme.ts

```diff
@@ -16,6 +16,10 @@ export const getMantineThemeOverride = (overrides?: {
     // Without it things look a little darker than before.
     black: '#111418',
 
+    colors: {
+        offWhite: ['#FDFDFD'],
```

Comment:
when I tried to override `white` it impacted a lot of other parts of the app.

Reply:
oh good, I think white should stay as is at least for now